api/envoy/extensions/quic/v3/proof_source.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package envoy.extensions.quic.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.quic.v3";
+option java_outer_classname = "ProofSourceProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: QUIC proof source config]
+// [#extension: envoy.quic.filter_chain_proof_source]
```

Comment:
I think the extension name should start with `envoy.quic.proof_source` (the category defined above).